src/main/java/net/atos/zac/app/planitems/PlanItemsRESTService.java

```diff
@@ -64,7 +64,7 @@
 import net.atos.zac.shared.helper.OpschortenZaakHelper;
 import net.atos.zac.util.DateTimeConverterUtil;
 import net.atos.zac.util.UriUtil;
-import net.atos.zac.zoeken.IndexeerService;
+import net.atos.zac.zoeken.IndexingService;
```

Comment:
While you're at it, could also consider renaming the `zoeken` package to `search`?

Reply:
Not everything is converted, so better keep it still